unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -173,9 +173,31 @@ public void SetMassProperties(string objectId, float mass, float drag, float ang
             return;
         }
 
-        public bool isStanding() {
-            return (m_Camera.transform.localPosition - standingLocalCameraPosition).magnitude
-                < 0.1f;
+        public bool? isStanding() {
+            //default to not standing if this isn't literally the PhysicsRemoteFPSAgentController
+            //this means the metadat for isStanding should always be false for all derived classes
+            if (this.GetType() != typeof(PhysicsRemoteFPSAgentController)) {
+                return null;
+            }
+
+            //if camera is in neither the standing or crouching predetermined locations, return null
+            if (
+                UtilityFunctions.ArePositionsApproximatelyEqual(
+                    m_Camera.transform.localPosition,
+                    standingLocalCameraPosition
```

Comment:
Would be good to pass in the epsilon here, as epsilons are per use basis rather than bake it in the function

Reply:
ok i've updated it so we can pass in a specific epsilon, but it will default to Unity's Vecto3.kEpsilon

Reply:
done!